module/functions/Update-TokenizedFiles.Tests.ps1

```diff
@@ -0,0 +1,45 @@
+# <copyright file="Update-TokenizedFiles.Tests.ps1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+
+$here = Split-Path -Parent $PSCommandPath
+$sut = (Split-Path -Leaf $PSCommandPath) -replace ".Tests"
+
+. "$here\$sut"
+
+$defaultTokenRegexPattern = "\#\{.*\}\#"
```

Comment:
I don't know if it matters, but this greedy `.*` followed by a specific terminator sequence can be problematic. For example, although this:

> Testing #{tag1}# a single tag.

matches OK

> Testing **#{tag1}#** a single tag.

(If the bold text isn't very clear, the matched text is `{tag1}#`, which is what we want.)

This input:

> Testing #{tag1}# and #{tag2}# a line with two tags.

will actually match from the start of the first tag to the end of the second tag:

> Testing **#{tag1}# and #{tag2}#** a line with two tags.

So in this case, the matched text is `#{tag1}# and #{tag2}#`. We've picked up both tags and all the text in between them. That might not be what you meant.

Consider `\#\{.*?\}\#` instead, which is the non-greedy match.